ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -705,4 +705,7 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
       stmnt.close()
     }
   }
+
+  override def checkDatabaseAvailable(connection: Connection): Unit =
+    assert(SQL"SELECT 1".as(get[Int](1).single)(connection) == 1)
```

Comment:
What log output will this produce if the database is not available? The output of the `assert` doesn't look useful. If if even can fail at all - when would `SELECT 1` not return `1`?

Reply:
log: if this fails the VerifiedDataSource will produce logs, that it attempts to connect to the db
select 1: with this we can make sure a full roundtrip between db and app, like a sanity check: testing all layers. I'd like to keep  it in if you don't mind.